src/test/scala/org/broadinstitute/dsde/firecloud/service/SubmissionServiceNegativeSpec.scala

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import org.broadinstitute.dsde.firecloud.FireCloudConfig
+import org.broadinstitute.dsde.firecloud.mock.MockWorkspaceServer
+import spray.http.HttpMethods.{GET, DELETE, POST}
+
+/**
+  * We don't create a mock server so we can differentiate between methods that get passed through (and result in
+  * InternalServerError) and those that aren't passed through at the first place (i.e. not 'handled')
+  */
+final class SubmissionServiceNegativeSpec extends ServiceSpec with SubmissionService {
+
+  def actorRefFactory = system
+
+  val localSubmissionsPath = FireCloudConfig.Rawls.submissionsPath.format(
+    MockWorkspaceServer.mockValidWorkspace.namespace,
+    MockWorkspaceServer.mockValidWorkspace.name)
+
+  val localSubmissionIdPath = FireCloudConfig.Rawls.submissionsIdPath.format(
+    MockWorkspaceServer.mockValidWorkspace.namespace,
+    MockWorkspaceServer.mockValidWorkspace.name,
+    MockWorkspaceServer.mockValidId)
+
+  "SubmissionService" - {
+    "non-GET requests hitting the /submissions/queueStatus path are not passed through" in {
+      allHttpMethodsExcept(GET) foreach { method =>
+        checkIfPassedThrough(routes, method, "/submissions/queueStatus", toBeHandled = false)
+      }
+    }
+
+    "non-GET/POST requests hitting the /workspaces/*/*/submissions path are not passed through" in {
+      allHttpMethodsExcept(GET, POST) foreach { method =>
+        checkIfPassedThrough(routes, method, localSubmissionsPath, toBeHandled = false)
+      }
+    }
+
+    "non-POST requests hitting the /workspaces/*/*/submissions/validate path are not passed through" in {
+      // Also excluding DELETE and GET since the path matches
+      // /workspaces/*/*/submissions/{submissionId} APIs as well
+      allHttpMethodsExcept(DELETE, GET, POST) foreach { method =>
+        checkIfPassedThrough(routes, method, s"$localSubmissionsPath/validate", toBeHandled = false)
```

Comment:
Submission IDs are UUIDs of form 8-4-4-4-12, I wonder if there's a way to reject `"validate"` as a `submissionID` on that basis

Reply:
Good idea. Is it documented somewhere that `submissionId`s are UUIDs?

Reply:
I don't know about documented, but I'm pretty sure this is where they come from: https://github.com/broadinstitute/rawls/blob/develop/core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala#L1463

Though Orchestration tests may need to be updated: https://github.com/broadinstitute/firecloud-orchestration/blob/develop/src/test/scala/org/broadinstitute/dsde/firecloud/mock/MockWorkspaceServer.scala#L38

Reply:
Thanks for the pointer! 

I don't feel comfortable changing the code based on the implementation.

Reply:
I've filed [GAWB-2807](https://broadinstitute.atlassian.net/browse/GAWB-2807) to keep track of the `submissionId` issue.

Reply:
Can you put a reference to this ticket in the code as well?